docs/admin-manual/configuration-macros.rst

```diff
@@ -11274,10 +11274,10 @@ has.
     in this directory are read, to define the metrics. The default
     directory ``/etc/condor/ganglia.d/`` is used when not specified.
 
-*condor_annex* Configuration File Macros
------------------------------------------
+:tool:`condor_annex` Configuration File Macros
+----------------------------------------------
```

Comment:
```suggestion
condor_annex Configuration File Macros
----------------------------------------
```

This reference in the subsection header is causing things to be wonky. probably better without any reference